portfolio/src/main/webapp/style.css

```diff
@@ -44,26 +42,74 @@
   display: block;
 }
 
-#funfacts {
+#frontcard {
+  width: 100%;
+  height: 500px;
+  background: rgba(0, 0, 0, .1);
+}
 
+#front-text {
+  bottom: 0;
+  height: 35%;
+  left: 0;
+  margin: auto;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 50%;
+}
+
+#funfacts {
+  top: 210px;
 }
 
 #greeting-container {
   margin-top: 20px;
 }
 
 #interests {
+  top: 140px;
+}
+
+#main-content {
+  background: url("./images/code.png") top/cover no-repeat fixed;
+  min-height: 500px;
+}
 
+.main-text {
+  background: #eee;
+  font-size: 20px;
+  height: auto;
+  padding: 0 50px 0 50px;
+  width: auto;
 }
 
 #mybackground {
+  top: 70px;
+}
+
+.sections {
+  background-color: #777;
+  height: 70px;
+  position: -webkit-sticky;
+  position: sticky;
+  width: 100%;
+}
 
+.sections h1 {
+  font-family: Arial, Helvetica, sans-serif;
```

Comment:
[use the shorthand `font` property](https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties), here and below